packages/minecraft.rb

```diff
@@ -13,6 +13,7 @@ class Minecraft < Package
 
   depends_on 'libx11'
   depends_on 'jdk8'
+  depends_on 'jdk17'
```

Comment:
We can only have one jdk requirement.

Reply:
1.16 and below need Java 8, so unless it's feasible to rework your setup to include both, I'd say switching to 17 and dropping support for older versions is the way to go

Reply:
Have you tried removing the jdk8 dependency from this package yet?  If so, just remove the `depends_on 'jdk8'` line and submit it with this pull request.

Reply:
Nope, I actually don't currently have a Chromebook readily available for me to test with until the end of the summer - hence why I wanted to avoid making any possibly breaking changes like that.

Reply:
Let's wait to make any changes on the Minecraft package until # #7097 is fixed. after that it will likely be that we can have multiple java versions installed concurrently

Reply:
@saltedcoffii looks like #7097 is now a won't fix, right?

Reply:
Where does that leave us on this PR, then? I've got an old Samsung Chromebook in my hands now, so I can run tests as needed once I get this thing set up.

Reply:
I appreciate this great package.
It works very good jdk8+1.165 version perfect.
Soddium Mod also works good
Minecraft cannot be installed without jdk8
I removed jdk8 and jdk17
But it doesnt work on 1.192 version

Reply:
Would `depends_on 'openjdk17'` work?

Reply:
I've switched both Java packages to their openjdk variants.

Reply:
Minecraft always crash because of conflict java
Even I remove jdk8 and install jdk17,
It still keep jdk8 environment.
I hope update to jdk17 dependencies.
If I can remove jdk8 completely, it will work.

I will try openjdk Thanks

Reply:
Marking as resolved per b12e215